scripts/dynamicmodal.js

```diff
@@ -0,0 +1,188 @@
+    imageLink: 'desk/SnapshootPortfolio7.png',
+    tags: ['HTML', 'CSS', 'Javascript'],
+    role: 'Back End Dev',
+    company: 'Canopy',
+    year: '2015',
+    sourceLink: '',
+    demoLink: '',
+  },
+];
+
+(() => {
+  projectSectionDom.innerHTML = '';
+  for (let i = 0; i < projectsArray.length; i += 1) {
+    let techlist = '';
+    for (let j = 0; j < projectsArray[i].tags.length; j += 1) {
+      techlist = `${techlist}<li class="tag">${projectsArray[i].tags[j]}</li>`;
+    }
+    projectSectionDom.innerHTML = `${projectSectionDom.innerHTML}
+    <ul class="${projectsArray[i].card}">
+    <li class="project">
+      <img src="${projectsArray[i].imageLink}" alt="Snapshot">
+        <div class="description">
+            <h2 class="title">${projectsArray[i].title}</h2>
+            <div class="info">
+                <p class="subTitle">${projectsArray[i].company}</p>
+                <ul class="infoContainer">
+                    <li class="role">${projectsArray[i].role}</li>
+                    <li class="year">${projectsArray[i].year}</li>
+                </ul>
+            </div>
+            <p class="projectDescription">
+                ${projectsArray[i].description}  
+            </p>
+            <div class="tags">
+                <ul class="tagsFlex">
+                  ${techlist}
+                </ul>
+            </div>
+            <button type="button" class="viewProject prj-view-btn">
+                See Project
+            </button>
+        </div>
+    </li>
+</ul>`;
+  }
+  const viewBtn = document.querySelectorAll('.prj-view-btn');
+  for (let i = 0; i < viewBtn.length; i += 1) {
+    viewBtn[i].addEventListener('click', (event) => {
+      const i = event.currentTarget.index;
+      nameDom.innerHTML = projectsArray[i].title;
+      roleNameModal.innerHTML = projectsArray[i].role;
+      yearDateModal.innerHTML = projectsArray[i].year;
+      descriptionDom.innerHTML = projectsArray[i].modaldesc;
+      projectImageLinkDom.src = projectsArray[i].imageLink;
+      sourceLinkDom.action = projectsArray[i].sourceLink;
+      demoLinkDom.action = projectsArray[i].demoLink;
+      technologiesDom.innerHTML = '';
+      for (let j = 0; j < projectsArray[i].tags.length; j += 1) {
+        technologiesDom.innerHTML = `${technologiesDom.innerHTML}<li class="tag">${projectsArray[i].tags[j]}</li>`;
+      }
+      popupModal.classList.remove('vanish');
+      popupModal.classList.add('show');
+      mobileMenu.classList.add('blur');
+      logo.classList.add('blur');
+    });
+    viewBtn[i].index = i;
+  }
+})();
+
+closeModal.addEventListener('click', () => {
+  popupModal.classList.remove('show');
+  popupModal.classList.add('vanish');
+  mobileMenu.classList.remove('blur');
```

Comment:
- [ ] Kindly also ensure the mobile view matches the design on [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

### Your Design
![image](https://user-images.githubusercontent.com/43995830/157838730-16c81b40-6b3e-4245-b96e-409b67e64b6c.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/157838822-fab9b478-70c3-457d-b3b6-1e2372580063.png)

Reply:
As requested by the previous code reviewer, kindly ensure mobile view matches the design on [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

Your Implementation 
![image](https://user-images.githubusercontent.com/10332499/157865714-34468fd9-3b47-4d10-84a5-8d909413c022.png)

Reply:
@micheaol Please see the screenshots I've provided, I think the desktop and mobile versions are now perfect. 
Desktop version
![Desktop](https://user-images.githubusercontent.com/73285210/157868116-249c3699-2ef9-425e-b74e-fd3baa999d1f.png)
Mobile version
![Mobile](https://user-images.githubusercontent.com/73285210/157868154-3499aa37-d269-4aed-87d8-4d8e4d62e07c.png)

Reply:
@shahedakhatoon ,

Kindly re-submit for review on your dashboard

Reply:
@micheaol okay, thank you.